save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/SettingsView.kt

```diff
@@ -93,24 +92,11 @@ typealias FieldsStateSetter = StateSetter<SettingsInputFields>
  * `Props` retrieved from router
  */
 @Suppress("MISSING_KDOC_CLASS_ELEMENTS")
-external interface SettingsProps : PropsWithChildren {
-    /**
-     * Currently logged-in user or null
-     */
-    var userInfo: UserInfo?
-
+external interface SettingsProps : UserInfoAwareMutablePropsWithChildren {
```

Comment:
Actually, I do not like the idea of extending plain classes and making inheritance on properties. 
Properties are simple structures aligned to a specific view, usually they should not have a common part. In my opinion - if needed, we can have common properties for views with multiple FC in it, but it also should be plain.

This particular inheritance can cause copy-paste issues for new people who will be simply copying properties and inheritance on classes and bring userInfo setter all over the code. 

I do not have strong objection about this change, just a note.